eZ/Bundle/EzPublishRestBundle/Tests/Functional/ContentTest.php

```diff
@@ -544,7 +634,8 @@ private function createVersionTranslation($restContentVersionHref, $languageCode
      */
     private function ensureLanguageExists($languageCode, $languageName)
     {
-        self::markTestIncomplete('@todo: Implement EZP-21171');
```

Comment:
Why not use `self::markTestIncomplete()` ? It doesn't make the suite fail. If it is not relevant, it might as well be removed.

Reply:
> Why not use self::markTestIncomplete() ? It doesn't make the suite fail. If it is not relevant, it might as well be removed.

It is no longer needed, for now missing language is added via symfony command in `.travis.yml` (changes for this are also included in this PR, see cea5e48 and 2672f9e) - so these tests actually work :)

Reply:
Then we may as well remove the method altogether, may we not ?

Reply:
> Then we may as well remove the method altogether, may we not ?

Yes/No/Maybe :)
Once we have proper Languages management via REST I planned here to put some REST calls to check if Language exists and create it if it doesn't.
So I could remove it for now, but then I'd need to add it once again. But I could if you want it gone for now :)

Reply:
Then if there is a todo item that you can track, let it be one :)